--- usr/share/applications/bitcoin-qt.desktop	2012-09-12 12:43:25.000000000 +0200
+++ ../bitcoin-qt.desktop	2012-09-12 16:07:01.524189674 +0200
@@ -4,7 +4,7 @@
 Comment=Bitcoin P2P Cryptocurrency
 Comment[fr]=Bitcoin, monnaie virtuelle cryptographique pair à pair
 Comment[tr]=Bitcoin, eşten eşe kriptografik sanal para birimi
-Exec=/usr/bin/bitcoin-qt
+Exec=/usr/share/privcoin/bitcoinqt-wrapper.sh
 Terminal=false
 Type=Application
 Icon=/usr/share/pixmaps/bitcoin80.xpm
